.github/workflows/l2geth_ci.yml

```
@@ -21,11 +22,14 @@ jobs:
       uses: actions/setup-go@v2
       with:
         go-version: 1.21.x
+
     - name: Checkout code
       uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)